lib/Service/ParticipantService.php

```diff
@@ -605,6 +605,29 @@ public function changeInCall(Room $room, Participant $participant, int $flags):
 		}
 	}
 
+	public function updateCallFlags(Room $room, Participant $participant, int $flags): void {
+		$session = $participant->getSession();
+		if (!$session instanceof Session) {
+			return;
+		}
+
+		if (!($session->getInCall() & Participant::FLAG_IN_CALL)) {
+			throw new \Exception('Participant not in call');
+		}
+
+		if ($flags === Participant::FLAG_DISCONNECTED || !($flags & Participant::FLAG_IN_CALL)) {
```

Comment:
```suggestion
		if (!($flags & Participant::FLAG_IN_CALL)) {
```
🤔 Can't contain `Participant::FLAG_IN_CALL` when it's disconnected anyway

Reply:
Of course :facepalm: